From 2418bc0d7cd770df124ce9aea3c86ebf7998393d Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Tue, 19 May 2015 11:03:55 +0800
Subject: [PATCH] MLK-10920 ARM: imx_v7_defconfig: enable MICREL phy

Enable CONFIG_MICREL_PHY for MICREL PHY since i.MX6ul boards install
micrel ksz8081 phy.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/configs/imx_v7_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index 8e80173..c8dc9f0 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -146,6 +146,7 @@ CONFIG_SMC91X=y
 CONFIG_SMC911X=y
 CONFIG_SMSC911X=y
 # CONFIG_NET_VENDOR_STMICRO is not set
+CONFIG_MICREL_PHY=y
 CONFIG_USB_PEGASUS=m
 CONFIG_USB_RTL8150=m
 CONFIG_USB_RTL8152=m
-- 
1.8.0

